examples/chip-tool-darwin/commands/common/CHIPCommandStorageDelegate.h

```diff
@@ -2,7 +2,7 @@
 #import <Foundation/Foundation.h>
 
```

Comment:
Should we add a NS_ASSUME_NONNULL_BEGIN here?

Reply:
Done.